### DISCUSSION TITLE
NoFail modifier variation, choose how many lives to play with. Kind of like the Easy modifier

### DISCUSSION THREAD

**Discussion by BallHitter10** [ppy/osu#27757] [Ideas]
I have been playing ezmod, and have especially enjoyed trying to get passes on difficult maps with it. Now when I play nomod I'm a little disappointed there is no option in between regular play passing (high challenge) and NoFail (No Challenge).
What do you all think? Is it worth adding a mod that gives extra but not infinite lives separate from EasyMod?

**Comment by Walavouchey** [comment]
was also suggested in a separate discussion https://github.com/ppy/osu/discussions/13857#discussioncomment-991932
